### DISCUSSION TITLE
India documentary channels

### DISCUSSION THREAD

**Discussion by adilshoukat** [iptv-org/community#1113] [Q&A]
The Indian documentary channels are missing. Please at least add the following famous ones.

- Animal Planet
- BBC Earth
- Nat Geo Wild
- National Geographic
- Discovery
- Travel XP
- History 18
- TLC
- EPIC

Note: I have watched all of them in Hindi audio before. So, I hope they can be added in Hindi audio in India category.

**Comment by eltoncomp** [comment]
Animal planet, discovery and tlc cannot be added because they are on the blocklist.

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
